node/src/components/small_network.rs

```diff
@@ -563,8 +625,32 @@ where
 
     /// Returns the set of connected nodes.
     #[cfg(test)]
-    pub(crate) fn connected_nodes(&self) -> HashSet<NodeId> {
-        self.outgoing.keys().cloned().collect()
+    pub(crate) fn outgoing_connected_nodes(&self) -> HashSet<NodeId> {
```

Comment:
I'm not sure I see the reason to exclude asymmetric connections from the `connected_nodes` getter(s).  Can you explain?

Reply:
I did you mean connections in the `blocklist`? I exclude them from these results to say "these are the connections that I have that are valid". That way, if a connection has been added to the `blocklist`, it is not counted towards a "complete" network in the test.